spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -0,0 +1,256 @@
+     - along:
+            type: s32
+            units: 0.4 mm
+            desc: Orbit along delta correction
+     - cross:
+            type: s32
+            units: 0.4 mm
+            desc: Orbit along delta correction
+     - dot_radial:
+            type: s32
+            units: 0.001 mm/s
+            desc: Velocity of orbit radial delta correction
+     - dot_along:
+            type: s32
+            units: 0.004 mm/s
+            desc: Velocity of orbit along delta correction
+     - dot_cross:
+            type: s32
+            units: 0.004 mm/s
+            desc: Velocity of orbit cross delta correction
+     - c0:
+            type: s32
+            units: 0.1 mm
+            desc: C0 polynomial coefficient for correction of broadcast satellite clock
+     - c1:
+            type: s32
+            units: 0.001 mm/s
+            desc: C1 polynomial coefficient for correction of broadcast satellite clock
+     - c2:
+            type: s32
+            units: 0.00002 mm/s^-2
+            desc: C2 polynomial coefficient for correction of broadcast satellite clock
+ - CodeBiasesPackedContent:
+     short_desc: SSR code biases corrections for a particular satellite.
+     desc: |
+      Code biases are to be added to pseudorange.
+      The corrections are conform with typical RTCMv3 MT1059 and 1065.
+     fields:
+     - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier (16 bit)
+     - value:
+            type: s16
+            units: 0.01 m
+            desc: Code bias value
+ - PhaseBiasesHeader:
+    short_desc: Header for phase biases message.
+    desc: Header of a SSR phase biases correction message.
+    fields:
+     - time:
+            type: GPSTime
+            desc: GNSS reference time of the correction
+     - update_interval:
+            type: u8
+            units: s
+            desc: Update interval between consecutive corrections
+     - iod_ssr:
+            type: u8
+            desc: |
+                IOD of the SSR correction. A change of Issue Of Data
+                SSR is used to indicate a change in the SSR 
+                generating configuration
+     - dispersive_bias:
+            type: u8
+            desc: |
+                Indicator for the dispersive phase biases property.
+     - mw_consistency:
+            type: u8
+            desc: |
+                Consistency indicator for Melbourne-Wubbena linear combinations
+     - n_corr:
+            type: u8
+            desc: |
+                Total number of ssr phase biases corrections.
+                First nibble is the size of the sequence (n),
+                second nibble is the zero-indexed counter 
+                (ith packet of n)
+ - PhaseBiasesValues:
+     short_desc: SSR phase biases corrections for a particular satellite.
+     desc: |
+      Code biases are to be added to pseudorange.
+      The corrections are conform with typical RTCMv3 MT1059 and 1065.
+     fields:
+     - code:
```

Comment:
I might be missing something - does the GnssSignal field in PhaseBiasesPackedContent not handle the code already? Or is a signal's code not material and that a given signal will have multiple codes. Would it make more sense then to have just sat?